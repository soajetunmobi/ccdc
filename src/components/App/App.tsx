import React, { FC } from 'react'
import { BrowserRouter } from 'react-router-dom'
import { AppLayout } from '../AppLayout/'
import Routes from 'routing/Routes'

export const App: FC = () => {
  return (
    <BrowserRouter>
      <AppLayout>
        <Routes />
      </AppLayout>
    </BrowserRouter>
  )
}
