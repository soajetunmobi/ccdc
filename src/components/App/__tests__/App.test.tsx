import React from 'react'
import { App } from '../App'
import { render } from '@testing-library/react'

test('it renders App correctly', () => {
  const { container } = render(<App />)
  expect(container).toMatchSnapshot()
})
