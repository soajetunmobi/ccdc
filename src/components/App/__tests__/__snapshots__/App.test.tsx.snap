// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders App correctly 1`] = `
<div>
  <div
    class="header"
  >
    <div
      class="logo"
    >
      <span
        class="cursor--pointer"
        data-testid="logo-link"
      >
        CCDC
      </span>
    </div>
  </div>
  <main
    class="content"
  >
    <div>
      <div
        class="loader"
      />
    </div>
  </main>
  <footer
    class="footer"
  >
    <span>
      CCDC
    </span>
    <br />
    2021
  </footer>
</div>
`;
